src/handlers.js

```diff
@@ -9,6 +9,7 @@ actionHandlers['!alert'] = {
     handle: (context, textContent) => {
         const formattedText = popup.formatEmotes(textContent, context.emotes, true).substr(7);
         popup.showText(formattedText, alertBg);
+        if (isAlertSound) new Audio(alertSoundFile).play();
```

Comment:
Can we use the long form if statement? they are far easier to read for people newer to coding

Reply:
Done!